composable/useFormat.ts

```diff
@@ -0,0 +1,25 @@
+export function useFormat() {
+  const formatCurrency = (value: number, currency: string = 'USD', options?: Intl.NumberFormatOptions): string => {
+    const isInteger = value % 1 === 0
+
+    const defaultCurrencyOptions: Intl.NumberFormatOptions = {
+      style: 'currency',
+      currency,
+      minimumFractionDigits: 2,
+      maximumFractionDigits: 2,
+    }
+
+    if (isInteger) {
+      defaultCurrencyOptions.minimumFractionDigits = 0
+      defaultCurrencyOptions.maximumFractionDigits = 0
+    }
+
+    const finalOptions = { ...defaultCurrencyOptions, ...options }
+
+    return new Intl.NumberFormat('en-US', finalOptions).format(value)
```

Comment:
Why is the locale hard coded here?

Reply:
Because we still haven't decided how we want to handle displaying usd vs euro

Reply:
Whats to decide? Every listing contains the currency, and if its not present its € since we are an EU company 
![image](https://github.com/user-attachments/assets/61625c5d-ffe4-4bb2-a329-8f9885e85b33)